tests/fakeoicsrv.py

```diff
@@ -215,7 +216,7 @@ def userinfo_endpoint(self, data):
     def registration_endpoint(self, data):
         try:
             req = self.parse_registration_request(data, "json")
-        except ValueError:
+        except DecodeError:
```

Comment:
as above, might raise ValueErrors as well for malformed content after sucessful json decoding.

Reply:
I am not going to fix this as I want to drop it. See #682